MediaBrowser.Providers/MediaInfo/AudioImageProvider.cs

```diff
@@ -137,9 +137,7 @@ public bool Supports(BaseItem item)
                 return false;
             }
 
-            var audio = item as Audio;
-
-            return audio != null;
+            return item is Audio audio;
```

Comment:
```suggestion
            return item is Audio;
```

Don't need to store this